template/src/pageComponents/buy-me-coffee/MemoCard.tsx

```diff
@@ -20,6 +21,7 @@ function MemoCard({ userName, message, userAddress, time }: CoffeeMemo) {
           <div className="flex items-start gap-1">
             <span className="text-3 text-bold w-fit whitespace-nowrap font-bold text-boat-color-palette-foreground">
               {userName}
+              {twitterHandle ? ` (@${twitterHandle})` : null}
```

Comment:
One more thing to reflect, is the power of ENS.

Ideally at some point we want to get to the point that a User should not share their Name or even their Twitter account.

BUT just by using their wallet we know their ENS, and from their ENS we know their Twitter account.

Also that will means we will get to a point that we will only allow users with ENS to use the contract.

Reply:
Again, food for thoughts for future PRs.

Reply:
Good thinking. Will think about this for follow ups in the future.